### ISSUE TITLE
link your minehut and Discord account to get credits

### ISSUE THREAD

**Issue by superJK92iscool** [Minehut/Meta#420]
### Feature request

**Checklist**
- [X] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [X] This is not a bug report, support request or plugin request
- [X] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [X] This issue does not contain any sensitive information 

**Platform**
 Website

**Describe the feature**
 I think you should be able to link your minehut and Discord account to get credits

**How the feature is useful**
 Everyone with a discord account. To make them happy

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Blocksnmore** [comment]
I feel this issue is just people wanting more free credits. Also it would be unfair to people who don't have/use discord

**Comment by Sniped** [comment]
I think the idea of linking your account to discord in-game would be better rather than just linking your account to your panel for free credits. There's no real benefit to doing so either apart from just the credits.

**Comment by Blocksnmore** [comment]
The only reason linking would really be done would be for roles on the discord but that's already got some staff saying it might not happen

**Comment by not-3than** [comment]
While this could be useful if player's are blacklisted from Minehut, it just seems like a cheap way to try and get more free credits.

**Comment by superJK92iscool** [comment]
Then have it so they get like 5 credits and you can link a discord account once and also discord is free

**Comment by MrTibo1** [comment]
> Then have it so they get like 5 credits and you can link a discord account once and also discord is free

Yes discord is free, but it's also 13+ so that would mean players under 13 will not be able to use this feature.

**Comment by superJK92iscool** [comment]
TBH I wanted it because I think there should be more ways to get credits then just voting.

**Comment by killergotrekt** [comment]
> TBH I wanted it because I think there should be more ways to get credits then just voting.

You are always free to purchase more credits in the shop.

**Comment by superJK92iscool** [comment]
> > TBH I wanted it because I think there should be more ways to get credits then just voting.
> 
> 
> 
> You are always free to purchase more credits in the shop.
> 
> 

I am 13 and my mom doesn't let me buy stuff unless it will help me and my mom

**Comment by orvit-png** [comment]
> > > TBH I wanted it because I think there should be more ways to get credits then just voting.
> > 
> > 
> > You are always free to purchase more credits in the shop.
> 
> I am 13 and my mom doesn't let me buy stuff unless it will help me and my mom if it cost irl money

I mean this in the nicest way possible, but that is not our problem.

**Comment by LukasmanMHdude** [comment]
I personally would want say 100 free credits for discord -> ingame

**Comment by honkling** [comment]
You already get this. From voting. Is 25 free credits DAILY not enough? People always suggest ideas like this not because it'll help improve Minehut but because it means they get free credits.

**Comment by superJK92iscool** [comment]
I forgot about that as well as how to vote for minehut
